Logger/FileLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace Logger
+{
+    class FileLogger : BaseLogger
+    {
+        private String? filePath;

```

Comment:
```suggestion
        private string? _FilePath;

```
You can simplify this to just use the C# keyword rather than the BCL name. You can refer to the guidelines on page 47 in chapter 2 of the book for more information about this.